packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3960,6 +3960,65 @@ void main() {
         tester.getRect(find.byKey(contentKey)).left + horizontalOffset,
       );
     });
+
+    group('The menu is attached to the bottom of the MenuAnchor content', () {
+      const double contentHeight = 100.0;
+      final MenuController menuController = MenuController();
+      final UniqueKey contentKey = UniqueKey();
+
+      Finder findMenuPanel() {
```

Comment:
What is your rationale for redefining a menu panel finder?

Reply:
On second look, it may be easier to just define 0 padding for the menu style and comparing the top of the first menu item, rather than looking up the menu panel. This is a nit.